src/main/webapp/resources/angular-templates/admin/partials/event/edit-event.html

```diff
@@ -50,24 +66,37 @@ <h3>Categories</h3>
             <div class="alert alert-danger" ng-if="event.ticketCategories.length === 0">
                 <i class="fa fa-warning"></i>
                 <span ng-if="canAddCategory(event.ticketCategories)">Please define at least one category</span>
-                <span ng-if="!canAddCategory(event.ticketCategories)">Please define the number of seats (Max Tickets)</span>
+                <span
+                    ng-if="!canAddCategory(event.ticketCategories)">Please define the number of seats (Max Tickets)</span>
             </div>
 
 
             <div class="row">
                 <div class="col-xs-12">
-                    <button type="button" class="btn btn-success" data-ng-disabled="!canAddCategory(event.ticketCategories)" data-ng-click="addCategory()"><i class="fa fa-plus"></i> Add new</button>
+                    <button type="button" class="btn btn-success"
+                            data-ng-disabled="!canAddCategory(event.ticketCategories)" data-ng-click="addCategory()"><i
+                        class="fa fa-plus"></i> Add new
+                    </button>
                 </div>
             </div>
 
-        <hr class="wMarginTop30px"/>
-        <div class="row">
-            <div class="col-xs-12 col-md-10 col-md-push-2">
-                <control-buttons data-form-obj="editEvent"></control-buttons>
-            </div>
-            <div class="col-xs-12 col-md-2 col-md-pull-10">
-                <button class="btn btn-lg btn-block btn-default" ng-click="reset()" type="button">Reset values</button>
+            <hr class="wMarginTop30px"/>
+
+            <!--        Adding terms and condition here-->
+            <input type="checkbox" required> I have read and agree to the terms and condition
```

Comment:
what's that code?

Reply:
Working on a different issues.

Reply:
If you work on a different issue, please do it in a separate branch, I can't accept a PR that fixes multiple issues.